tests/all-tests.html

```diff
@@ -11,6 +11,11 @@
 
     <script>
     QUnit.testSuites("The Chart Authoring Tool Tests", [
+        "./html/d3ViewComponent-Tests.html",
+        "./html/pie-Tests.html",
+        "./html/legend-Tests.html",
+        "./html/pieChart-Tests.html",
+        "./html/utils-Tests.html",
```

Comment:
If we keep the separate utility file, the test should probably come before dsViewComponent-Tests. It's better to have the dependencies run first.